src/pages/home/report/ReportActionsList.tsx

```diff
@@ -188,6 +188,7 @@ function ReportActionsList({
     const hasFooterRendered = useRef(false);
     const lastVisibleActionCreatedRef = useRef(report.lastVisibleActionCreated);
     const lastReadTimeRef = useRef(report.lastReadTime);
+    // Single MoneyRequest
```

Comment:
Do we need this comment?

Reply:
It is showing as a change, because ola removed it in her PR and this branch was based on it